infra/base-images/base-builder/precompile_centipede

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash -eu
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+echo -n "Precompiling centipede"
+
+# Build Centipede with bazel.
+cd "$SRC/centipede/"
+./install_dependencies_debian.sh
+bazel build -c opt :all
+
+apt uninstall curl gnupg apt-transport-https git bazel binutils libssl-dev
```

Comment:
Use apt-get not apt which is not meant for scripts.
However, I absolutely think you should not be uninstalling any of these. All of these are used elsewhere.

Reply:
What would be a good way for me to check which dependencies are used elsewhere?
I felt very unsafe when I added this line, but not sure if there is a list of dependencies for me to check through.